### DISCUSSION TITLE
Where does deno.land host its assets, websites, apis, etc.?

### DISCUSSION THREAD

**Discussion by lancejpollard** [denoland/deno#19702] [Q&A] [answered]
The deno.land website used to be in this repo: [denoland/dotland](https://github.com/denoland/dotland), but now not sure where it is.

But my question is kind of a tangent question, just wondering what deno has chosen as the best platform for hosting its website and other "sites" or servers it uses. Does it use AWS, or Google Cloud (or something else)? CloudFlare, DigitalOcean, Azure, etc.?

Just wondering what such a central tool would choose as its main place for hosting. Sorry if it's tangential, but don't know where else to ask.

Personally, trying to decide between AWS and Google Cloud, after being a programmer for 15+ years. AWS is the legacy, but perhaps Google Cloud is the way to go these days. But I've heard (and experienced) Google Cloud being a nightmare in some cases, so really curious what deno has chosen.

Thanks anyways!

**Comment by willnewby** [comment, marked as answer]
We use both AWS and Google Cloud (for different things). They both have their strengths and weaknesses, depending on the use case. AWS typically exposes smaller pieces of a cloud system, which can be really helpful if you know exactly what you need and how you want it to work. Google Cloud tends towards a more "product-focused" cloud environment, meaning you get more complete solutions to the kinds of problems you have. In the case of Deno Deploy, we use both.
